src/robusta/integrations/receiver.py

```diff
@@ -209,50 +220,53 @@ def __validate_request(self, action_request: ExternalActionRequest, validate_tim
             logging.error(
                 f"Rejecting incoming request because it's too old. Cannot verify request {action_request}"
             )
-            return False
+            return ValidationResponse(http_code=500, error_code=4500, error_msg="Illegal timestamp")
 
         signing_key = self.event_handler.get_global_config().get("signing_key")
         body = action_request.body
         if not signing_key:
             logging.error(f"Signing key not available. Cannot verify request {body}")
-            return False
+            return ValidationResponse(http_code=500, error_code=4501, error_msg="No signing key")
 
         # First auth protocol option, based on signature only
         signature = action_request.signature
         if signature:
             generated_signature = sign_action_request(body, signing_key)
-            return hmac.compare_digest(generated_signature, signature)
+            if hmac.compare_digest(generated_signature, signature):
+                return ValidationResponse()
+            else:
+                return ValidationResponse(http_code=500, error_code=4502,  error_msg="Signature missmatch")
 
         # Second auth protocol option, based on public key
         partial_auth_a = action_request.partial_auth_a
         partial_auth_b = action_request.partial_auth_b
         if not partial_auth_a or not partial_auth_b:
             logging.error(f"Insufficient authentication data. Cannot verify request {body}")
-            return False
+            return ValidationResponse(http_code=500, error_code=4503, error_msg="Missing auth input")
```

Comment:
are those human readable message suppose to be 100% clear to users? Maybe add a section in the docs on how to respond to this errors. The link to the docs can appear in the UI as well.

Reply:
These are internal errors. 
I'm not sure the user has anything to do but contact us.
This error for example, doesn't have anything to do with what the user is doing